content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc

```diff
@@ -0,0 +1,78 @@
+
+Below you will find the guidelines and process for an Apple application developer to register an iOS app for approval that has completed the Yubico iOS SDK integration.
+
+NOTE: This process is not required for Android applications.
+
+=== Approval Process
+image::iOS_App_Review_Process.png[]
+The <<Actions Required By You,Actions Required By You>> section provides detailed instructions for each step in the approval process. Submitting an app to the App Store without the Yubico 5Ci MFi PPID will result in rejection from Apple.
+
+=== UI Components for YubiKey iOS Apps
+Download the https://github.com/Yubico/yubikit-ios[SDK.zip] file for a list of free-to-use example assets for use in your YubiKey-enabled iOS application. Under the extracted folder: 
+
+- Waiting for NFC scan Swift components can be found under: YubiKitDemo/YubiKitDemo/UI/OTP/Views/.
+
+- Lightning animation Swift components can be found under: YubiKitDemo/YubiKitDemo/UI/SharedUI/Lightning/.
+
+=== App Design and Functional Guidelines
+YubiKey-enabled iOS apps must be built using the latest version of the iOS SDK and conform to https://developer.apple.com/app-store/review/guidelines/[Apple’s iOS Developer Program guidelines].
+
+For approval, the app must meet the following guidelines:
+
+ - Stability: The app will not freeze, crash, rapidly drain battery, or put unnecessary strain on device resources. 
+ - User Experience: The app must communicate YubiKey status to users in an acceptable fashion determined by Yubico.
+   * Insert YubiKey 5Ci Animation Guideline
+    ** If a YubiKey has not yet been inserted while the app is in use, display the insert key animation.
+  image:insert_key_to_authenticate.png[]
+
+   * Touch YubiKey 5Ci Animation Guideline
+    ** If the YubiKey is inserted when the app requests an operation, display the touch key animation.
+  image:scan_your_yubikey.png[]
+
+   * Waiting for NFC Scan Guideline
+    ** If the app is waiting to communicate over NFC, display a “Ready to Scan” animation and image to guide the user to the appropriate location to hold the key near.
+  image:touch_key_to_complete_operation.png[]
+
+ - Quality: The app will provide a user experience that does not diminish the users’ perceived quality of the YubiKey and its performance.
+   * Test all features that interact with the YubiKey functionality before registering an app for review. See features below that may be verified by Yubico for quality if applicable.  
+ - Brand: The app must follow the Yubico usage guidelines when presenting the Yubico image or logo.
+   * See https://www.yubico.com/press/images/ for usage guidelines.
+
+
+=== [[Actions Required By You]]Actions Required By You
+
+1. Register App with Yubico - Register your app by filling out our https://partners.yubico.com/prm/English/c/lightning-partner-preview-support/[Yubico App Registration Form]. Include the following:
```

Comment:
Shouldn't this link to /Software_Projects/Mobile_SDK/Register_Your_App/?

Reply:
This is what was on the original doc spec, we would need clarity from Lan